### PR TITLE
Update conflicts documentation to include Swift 2 code blocks

### PR DESCRIPTION
**Pull request by rhyshort**
## What

- Add missing headers to `CloudantSync.h` to enable conflict resolution
- Add Swift 2 code blocks to conflicts documentation

## Reviewers

reviewer @mikerhodes 
reviewer @alfinkel

### CODE REVIEW

**Comment by alfinkel** [comment]
It might be clearer if we labeled the example code blocks with the language that the code is an example for.  Since now we have the Objective C and Swift examples coexisting in the docs.  Just a thought.  +1 otherwise.

**Comment by rhyshort** [comment]
Any idea on the labelling? I haven't manged to come up with a scheme I like

File: doc/conflicts.md [hunk 1]
```diff
@@ -120,6 +123,11 @@ for (NSString *docId in [datastore getConflictedDocumentIds]) {
     NSLog(@"%@", docId);
 }
 ```
+```swift
+for docId in datastore.getConflictedDocumentIds()  {
```
Comment (mikerhodes) [inline doc/conflicts.md]: too many spaces before the brace

File: doc/conflicts.md [hunk 2]
```diff
@@ -174,6 +198,16 @@ It is also possible to return a `CDTDocumentRevision` from
     return rev;
 }
 ```
+```swift
+class CDTMergeResolver : NSObject, CDTConflictResolver {
+    func resolve(docId: String!, conflicts: [AnyObject]!) -> CDTDocumentRevision! {
+        let rev = conflicts.first!
+        rev.body // ...update body, perhaps with data from the other conflicts
+        rev.attachments // ...you can also create/update/delete attachments
```
Comment (mikerhodes) [inline doc/conflicts.md]: Missing `=` on the two above lines.

I'd also prefer to see block comments as with the objc.

**Comment by mikerhodes** [comment]
+1, though you'll either want to squash out or correct the typos in 7738b47

**Comment by rhyshort** [comment]
I'll squash out, when @alfinkel is happy

**Comment by alfinkel** [comment]
Sorry, I missed your last message.  The way you are labeling the example code blocks looks good to me.  +1.

**Pull request merged**